rails_docker/docker-compose.test.yml

```diff
@@ -1,4 +1,4 @@
-version: '3.2'
+version: '3.5'
```

Comment:
@olivierobert better to move to the latest `3.6` version

Reply:
@abhinavmsra I wished but most CI servers are stuck to Docker `17.11` like Circle CI so Docker Compose `3.2`. Semaphore supports Docker `17.18` 🙌. It's possible to install a newer version of Docker but it's extra build step.